src/views/projects/Cob/Revision.svelte

```diff
@@ -149,6 +154,36 @@
       <span class="layout-desktop txt-small">
         {utils.formatTimestamp(revisionTimestamp)}
       </span>
+      {#if previousRevOid && previousRevId}
+        <Floating>
+          <svelte:fragment slot="toggle">
+            <SquareButton size="small">
+              <Icon name="ellipsis-small" />
+            </SquareButton>
+          </svelte:fragment>
+          <svelte:fragment slot="modal">
+            <Dropdown
+              items={[
+                { title: previousRevOid, value: previousRevOid, badge: null },
+              ]}
+              selected={previousRevOid}>
+              <svelte:fragment slot="item" let:item>
+                <ProjectLink
+                  title="{item.value}..{revisionOid}"
```

Comment:
added this `title` cause originally it was only showing the first part of the hash.